src/adapters/store.js

```diff
@@ -5,9 +5,11 @@ import ExtendedString from "../libs/string";
 import LocalStore from "../libs/local_store"
 import MasqStore from "../libs/masq"
 
-export default class Store {
+export default class Store extends EventTarget {
```

Comment:
Very nice !
It's an open subject, but i think composition is better here than heritance. Here store have nothing to do with Event. it's Like you build a House and herit from firetruck beacause you need red paint.

Reply:
done !